modules/opengl/src/shader/shader.cpp

```diff
@@ -324,6 +318,13 @@ void Shader::rebuildShader(ShaderObject *obj) {
                   "Id: " + toString(program_.id) + ", resource: " + obj->getFileName() +
                       " successfully reloaded",
                   LogLevel::Info, LogAudience::User);
+                  
+        auto log = utilgl::getProgramInfoLog(program_.id);
+        if (!log.empty()) {
```

Comment:
might cause some logging on Intel or AMD

Reply:
think it is better to log more then less in the case of OpenGL. If someone thinks it is to much then we can try an limit it